contrib/fwupd.spec.in

```diff
@@ -74,8 +74,7 @@ BuildRequires: pango-devel
 BuildRequires: cairo-devel cairo-gobject-devel
 BuildRequires: freetype
 BuildRequires: fontconfig
-BuildRequires: dejavu-sans-fonts
-BuildRequires: adobe-source-han-sans-cn-fonts
```

Comment:
Need to modify dependencies.xml for this change too to let CI find it.

On Ubuntu/Debian the package name is `fonts-noto-hinted` and on Arch it's `noto-fonts` IIRC

Reply:
Gahh, that was supposed to be in a different PR, will move and fix it. Thanks.